src/components/calcite-tip-manager/calcite-tip-manager.tsx

```diff
@@ -186,15 +188,17 @@ export class CalciteTipManager {
   };
 
   tipManagerKeyDownHandler = (event: KeyboardEvent): void => {
+    if (event.target !== this.container) {
```

Comment:
[Suggestion] I think you could avoid `container` by checking if `currentTarget#onkeyup === this.tipManagerKeyDownHandler` or using a data attribute and checking for it. Feel free to ignore.

Reply:
sand: I'd suggest blacklisting the element over whitelisting - this method would prevent the event from correctly firing if it bubbled up from any child elements, and I'm not sure that's ultimately desired.

Reply:
I think whitelisting is better than blacklisting. We only want the event to do something if it came from that node.